source/shared/cpp/ObjectModel/Enums.cpp

```diff
@@ -19,6 +19,7 @@ namespace AdaptiveSharedNamespace
             {AdaptiveCardSchemaKey::Attention, "attention"},
             {AdaptiveCardSchemaKey::BackgroundColor, "backgroundColor"},
             {AdaptiveCardSchemaKey::BackgroundImage, "backgroundImage"},
+            {AdaptiveCardSchemaKey::BackgroundImageMode, "mode"},
```

Comment:
>BackgroundImageMode [](start = 36, length = 19)

Since this is just for use to parse things back and forth from json, it would be okay to call this "Mode" and allow it to be reused if something else with a Mode comes along.